cartridge/issues.lua

```diff
@@ -337,6 +341,13 @@ local function list_on_instance(opts)
 end
 
 local function list_on_cluster()
+    local ret, err = lua_api_proxy.call(mod_name .. '.list_on_cluster')
+    if ret ~= nil then
+        return ret
+    elseif err ~= nil then
+        return nil, err
+    end
```

Comment:
Looks a bit tricky. If remote function returns "box.NULL" or nothing it is considered as need to perform a "local call".

Reply:
Fixed.